__init__.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+
+from flask import Blueprint
+
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, HERE)
+
+__version__ = "0.1"
+
+entsoe_data_bp = Blueprint("entsoe", __name__)
+
+DEFAULT_COUNTRY_CODE = "NL"
+DEFAULT_TIMEZONE = "Europe/Amsterdam"  # This is what we receive, even if ENTSO-E documents Europe/Brussels
```

Comment:
I didn't see mention of Amsterdam in the response snippets that I saw in our chat, but rather something like CEST. I guess this is fine, since Brussels and Amsterdam should always be in sync afaik.